app/models/statement.rb

```diff
@@ -24,15 +24,19 @@ def latest_reconciliation
   end
 
   def record_actioned!
-    self.actioned_at = DateTime.now
+    self.actioned_at = Time.now
     save!
   end
 
+  def actioned?
+    !self.actioned_at.nil?
+  end
```

Comment:
I think this method probably belongs in `StatementDecorator`, where similar methods like `done?`and `verified?` live.